app/jobs/onceoff/rebake_all_html_theme_fields.rb

```diff
@@ -0,0 +1,11 @@
+module Jobs
+  class RebakeAllHtmlThemeFields < Jobs::Onceoff
+    def execute_onceoff(args)
+      ThemeField.where(type_id: ThemeField.types[:html]).find_each do |theme_field|
+        theme_field.update(value_baked: nil)
```

Comment:
I intentionally picked `update` over `update!`, so that one failure will not throw off the entire job.